app/domain/authentication/authn_gcp/consts.rb

```diff
@@ -7,6 +8,8 @@ module AuthnGcp
     INSTANCE_NAME_RESTRICTION_NAME = "instance-name"
     SERVICE_ACCOUNT_ID_RESTRICTION_NAME = "service-account-id"
     SERVICE_ACCOUNT_EMAIL_RESTRICTION_NAME = "service-account-email"
+    PERMITTED_CONSTRAINTS = %w(authn-gcp/instance-name authn-gcp/project-id authn-gcp/service-account-id authn-gcp/service-account-email)
```

Comment:
do we really need the `authn-gcp/` prefix here? can't we add it later so this const looks nicer?

Reply:
something like this:
```
PERMITTED_CONSTRAINTS = %w(instance-name project-id service-account-id service-account-email)

prefixed_permitted_constraints = PERMITTED_CONSTRAINTS.map { |k| "authn-gcp/#{k}" }
```

or even
```
PERMITTED_CONSTRAINTS = %w(instance-name project-id service-account-id service-account-email)

prefixed_permitted_constraints = PERMITTED_CONSTRAINTS.map { |k| "#{AUTHN_PREFIX}#{k}" }
```

btw, you can use `authenticator_name` instead of `AUTHN_PREFIX`.
```
PERMITTED_CONSTRAINTS = %w(instance-name project-id service-account-id service-account-email)

prefixed_permitted_constraints = PERMITTED_CONSTRAINTS.map { |k| "#{authenticator_name}/#{k}" }
```

Reply:
Done :)